src/extension.ts

```diff
@@ -104,6 +105,16 @@ export async function activate(context: vscode.ExtensionContext) {
         })
         await ext.telemetry.start()
 
+        const extContext:ExtContext = {
```

Comment:
Nit: do we need to specify/import the interface? I think the compiler should be able to pick this up?